File: frontend/uvtdorms/src/app/interfaces/ticket-dto.ts
import { StatusTicket } from '../enums/status-ticket';
import { TipInterventie } from '../enums/tip-interventie';

export interface TicketDto {
  id: string;
  creationDate: string[];
  statusTicket: StatusTicket;
  tipInterventie: TipInterventie;
  title: string;
  description: string;
  alreadyAnuncement: boolean;
  studentEmail: string;
  roomNumber: string;
}

File: backend/src/main/java/com/uvtdorms/repository/dto/response/TicketDto.java
package com.uvtdorms.repository.dto.response;

import java.time.LocalDateTime;

import com.uvtdorms.repository.entity.enums.StatusTicket;
import com.uvtdorms.repository.entity.enums.TipInterventie;

import lombok.Data;

@Data
public class TicketDto {
    String id;
    LocalDateTime creationDate;
    StatusTicket statusTicket;
    TipInterventie tipInterventie;
    String title;
    String description;
    boolean alreadyAnuncement;
    String studentEmail;
    String roomNumber;
}
